From 577a81ec79ac1aa1cc8a45b122d2eb3c36051f78 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 14 Dec 2016 14:16:00 -0800
Subject: [PATCH] Revert "TOOLS-1636 updates-imgadm et al with node v4 will not
 flush stdout on exit" (causes jenkins build failures)

This reverts commit 37877d5c4eba6ec9c03678d2b4faaaa87011bb8d.
---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 69bd22e..f72a187 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "dependencies": {
     "manta": "3.1.3",
     "smartdc": "7.3.0",
-    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.3"
+    "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#38332b1"
   },
   "engines": {
     "node": ">=0.8"
-- 
2.21.0

